coalib/results/Diff.py

```diff
@@ -381,6 +381,19 @@ def add_lines(self, line_nr_before, lines):
         linediff.add_after = lines
         self._changes[line_nr_before] = linediff
 
+    def add_line(self, line_nr_before, line):
+        """
+        Adds line after the given line number.
+
+        :param line_nr_before: Line number of the line before the additions.
```

Comment:
yeah! I wrote complete function for `add_line` , but @Makman2 suggested to use the `add_lines` function. 
I've modified the ConflictError's to `line(s)`.
That will cover both the cases.